components/channel_selector_modal/channel_selector_modal.jsx

```diff
@@ -175,19 +178,6 @@ export default class ChannelSelectorModal extends React.Component {
 
         const buttonSubmitText = localizeMessage('multiselect.add', 'Add');
 
-        let channels = [];
-        if (this.props.channels) {
-            channels = this.props.channels.filter((channel) => {
-                return (
-                    (channel.delete_at === 0) &&
-                    (channel.scheme_id !== this.currentSchemeId) &&
-                    (this.props.alreadySelected.indexOf(channel.id) === -1) &&
```

Comment:
This will start to show already selected channels?

Reply:
Let me verify.

Reply:
To make this properly remove the already selected channels I think we'll need to add something like `not_associated_to_group` to `/api/v4/channels`.